src/base-lib/src/constant/constant.ts

```diff
@@ -397,36 +397,91 @@ export const EVENT_SOURCE_LOAD_DATA_FLOW = 'LoadDataFlow';
 
 export const EMR_ARCHITECTURE_AUTO = 'Auto';
 
-export const CLICKSTREAM_RETENTION_VIEW_PLACEHOLDER = 'Retention_View';
-export const CLICKSTREAM_SESSION_VIEW_PLACEHOLDER = 'Session_View';
-export const CLICKSTREAM_USER_DIM_VIEW_PLACEHOLDER = 'User_Dim_View';
+
+export const CLICKSTREAM_LIFECYCLE_VIEW_NAME = 'clickstream_lifecycle_view_v2';
```

Comment:
Let's move them to a single `dashboard` file for the constant of data modeling and dashboard.